pkg/controller/inferenceservice/resources/istio/virtualservice.go

```diff
@@ -237,8 +244,14 @@ func getPredictStatusConfigurationSpec(endpointSpec *v1alpha2.EndpointSpec, comp
 		return nil, PredictorSpecMissing
 	}
 
+	if componentStatusMap == nil {
```

Comment:
Big ask but:
Should we call StatusConfigurationSpec a ComponentStatus? I feeel like StatusSpec is an oxymoron.

Reply:
seems like can be a separate change?